docs/docs/integrations/libraries/dbt/dbt-component.md

```diff
@@ -1,10 +1,16 @@
 ---
-title: 'Dagster & dbt with components'
+title: dbt Component
```

Comment:
It seems like the other integrations the format is `title: {integration} (component)`